packages/@aws-cdk/cx-api/lib/cxapi.ts

```diff
@@ -106,6 +107,35 @@ export interface AssetMetadataEntry {
   s3KeyParameter: string;
 }
 
+export interface ContainerImageAssetMetadataEntry {
+  /**
+   * Type of asset
+   */
+  packaging: 'container-image';
+
+  /**
+   * Path on disk to the asset
+   */
+  path: string;
+
+  /**
+   * Logical identifier for the asset
+   */
+  id: string;
+
+  /**
+   * Name of the parameter that takes the repository name
+   */
+  repositoryParameter: string;
+
+  /**
+   * Name of the parameter that takes the tag
+   */
+  tagParameter: string;
+}
+
+export type AssetMetadataEntry = FileAssetMetadataEntry | ContainerImageAssetMetadataEntry;
```

Comment:
Good point. Doesn't actually matter as no user ever needs to call this (I hope).

@eladb, why don't we bundle `@aws-cdk/cx-api` as part of `@aws-cdk/cdk`?

Reply:
Why we we need to bundle?

Reply:
Because it's an implementation detail that there's a shared package with definitions between `aws-cdk` and `@aws-cdk/cdk`. Nobody else should need to see or interact with it. Therefore it does not need to be exposed over jsii. 

As a practical matter, not exposing over jsii means we can safely use TypeScript-isms that otherwise wouldn't be allowed.